docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/LoadingText.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic.DataOperations
+{
+    public static class LoadingText
+    {
+        // This examples shows all the ways to load data with TextLoader.
+        public static void Example()
+        {
+            // Create 5 data files to illustrate different loading methods.
+            var dataFiles = new List<string>();
+            var random = new Random();
+            var dataDirectoryName = "DataDir";
+            Directory.CreateDirectory(dataDirectoryName);
+            for (int i = 0; i < 5; i++)
+            {
+                var fileName = Path.Combine(dataDirectoryName, $"Data_{i}.csv");
+                dataFiles.Add(fileName);
+                using (var fs = File.CreateText(fileName))
+                    // Write random lines without header
```

Comment:
Could we have another example with header?

Reply:
this is an example for Load() method and all it's variants. header applies to creating the loader itself. I'll add that separately to CreateTextLoader() API. 

---
In reply to: [289105567](https://github.com/dotnet/machinelearning/pull/3793#discussion_r289105567) [](ancestors = 289105567)